sciLifeLab_utils/run_assemblies.py

```diff
@@ -89,54 +89,20 @@ def main(args):
             sample_files.remove(pair2_file)
             library += 1
         sample_YAML.close
-        if not hasattr(args, "email"):
-            args.email = None
-        submit_job(sample_YAML_name, args.global_config, sample_dir_name ,
-                pipeline, args.env, args.email, args.time, args.project,
-                args.threads, args.qos) # now I can submit the job to slurm
-        os.chdir(projectFolder)
 
+        #Run the job
+        all_modules = {
```

Comment:
Ah yes now I think I understand. [This](https://github.com/SciLifeLab/NouGAT/pull/28/files#diff-df356f484ebafb99845d8e144effa44dR102) will read what tools are defined in the sample config and add `module load` for whatever is defined in the `all_modules` dictionary. 

Ah yes. Right now bwa and samtools are hardcoded. I can make a check and add them only `if "align" in tools`